### PR TITLE
Various cleanups

### PR DESCRIPTION
**Pull request by hfiguiere**
This way the package is simpler and smaller.

The app starts but I get an openAL error, like the currently available package.

### CODE REVIEW

File: jp.yvt.OpenSpades.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
     "runtime-version": "21.08",
     "sdk": "org.freedesktop.Sdk",
     "command": "openspades",
+    "rename-desktop-file": "openspades.desktop",
```
Comment (jschlumpp) [inline jp.yvt.OpenSpades.json:7]: That's a pretty nice feature :+1:

**Review by jschlumpp — APPROVED** [review]

**Comment by jschlumpp** [comment]
> The app starts but I get an openAL error, like the currently available package.

What error? On what distribution did you encounter this problem (and did any noteworthy changes related to audio setup)? Can other flatpak applications output audio?

**Comment by hfiguiere** [comment]
I don't get the error anymore, for some reason

**Comment by hfiguiere** [comment]
nobody was interested in this and given that the manifest changed format just because....

**Pull request closed without merging**